src/App.svelte

```diff
@@ -1,7 +1,12 @@
 <script>
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
-  console.log(exampleData);
+  export let grabIP = function(text){
+    const ips = document.getElementsByClassName('ip');
+    const toCopy = Array.from(ips).filter((ip) => {return ip.value === text})[0];
+    toCopy.select();
```

Comment:
You should use find() instead of filter. 

also destructure the IP and just return the internal value ```find(({ value }) => value ==ip)``` or something

Reply:
find is a good idea, but the whole reason for not just returning the text is to highlight the text in the box. That's the whole point of this rewriting.

Reply:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard

Reply:
Find would return the node itself, unless i'm misunderstanding you

Reply:
yeah I changed it from filter to find, maybe I'm not understanding what you mean by destructing it beforehand.